os/keycodes.h

```diff
@@ -12,9 +12,11 @@
 
 #ifdef _WIN32
 #include "os/windows/keycodes.h"
-
-#else
+#endif
+#ifdef linux
 #include "os/x11/keycodes.h"
+#else
+#include "os/sdl/keycodes.h"
```

Comment:
Likewise, swap the Mac and Linux case here so that the Linux case is the else case.

Reply:
Makes sense fixed it

Reply:
Hmm, I still think it would make sense to keep it as an if-else chain with the X11 case as the `#else` case (without a reference to `__linux__`. This should make the X11 case trigger with other Unix-like systems that uses X11 (BSD etc.).